server/hydrogen-render/render-page-html.js

```diff
@@ -8,7 +8,7 @@ const safeJson = require('../lib/safe-json');
 const getDependenciesForEntryPointName = require('../lib/get-dependencies-for-entry-point-name');
 const getFaviconAssetUrls = require('../lib/get-favicon-asset-urls');
 
-async function renderPageHtml({
```

Comment:
This doesn't need to be async and we actually used this incorrectly in `server/routes/client-side-room-alias-hash-redirect-route.js` since we never awaited the promise before.

No need to be async though so remove `async`